graylog2-web-interface/src/contexts/DefaultQueryClientProvider.tsx

```diff
@@ -30,6 +31,11 @@ const defaultOptions = {
     queries: {
       refetchOnWindowFocus: false,
       networkMode: 'always' as const,
+      retry: (failureCount, error) => {
+        if (isNumber(error?.status) && (error.status >= 400) && (error.status <= 417)) return false;
```

Comment:
We should do this for all 4xx status codes. Also, due to the later comparisons, the check for `isNumber` is not needed. Therefore we can simplify this to:
```suggestion
        if (error?.status >= 400 && error?.status < 500) return false;
```